### PR TITLE
Add THESIAS 3.1

### PR DESCRIPTION
**Pull request by daissi**
Add a new recipe for THESIAS 3.1

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by daissi** [comment]
Hi @bioconda/core

I don't know what could be wrong here, do you have any advise?

Best,
Dylan

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@daissi You'll need to either change the flags given to the compiler or skip building on OSX, since `-soname` isn't a clang option.

**Comment by daissi** [comment]
> @daissi You'll need to either change the flags given to the compiler or skip building on OSX, since `-soname` isn't a clang option.

@dpryan79 thanks for your help. I have disabled the build on macos for now, until I fix the upstream makefile.

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

File: recipes/thesias/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% set version = "3.1" %}
+
+package:
+  name: thesias
+  version: {{ version }}
+
+source:
+  url: https://github.com/daissi/thesias/archive/bd558973415a3ab13e24df1204cc8443858318e8.tar.gz
```
Comment (dpryan79) [inline recipes/thesias/meta.yaml]: Can we use the tagged 3.1 release instead?

Comment (daissi) [reply recipes/thesias/meta.yaml]: The tagged 3.1 is only to provide an old (2007) archive with only binaries. I will release something like 3.1.1 as soon as I have fixed the makefile. It's why I am packaging THESIAS for bioconda and Debian to seen if everything is fine on different systems. For now, it is better to use git master with maybe something like 3.1.post1 if you prefer.

Comment (dpryan79) [reply recipes/thesias/meta.yaml]: Why not just tag 3.1.1 now then? Our bot will then automatically create new PR when you tag new releases.

Comment (daissi) [reply recipes/thesias/meta.yaml]: I have tagged 3.1.1 and updated the recipe accordingly.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by daissi** [comment]
@dpryan79 thanks!

It seems there is something wrong with the recipe.

When I tried to use THESIAS, I get this error:

`Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: GraficT has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0`

I am able to fix this error by install a newer openjdk (11 and not 8) from conda-forge. Should I increase the version of openjdk in the recipe?

**Comment by dpryan79** [comment]
Apparently that'll be needed, yes.

**Pull request merged**